resource-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/resource/kafka/KafkaInitializer.java

```diff
@@ -120,23 +120,23 @@ Materializer materializer(final ActorSystem system) {
     // Serialiser for upstream test input
     static class RequestSerializer implements Serializer<JsonNode> {
         @Override
-        public byte[] serialize(final String s, final JsonNode userRequest) {
+        public byte[] serialize(final String s, final JsonNode resourceRequest) {
             try {
-                return MAPPER.writeValueAsBytes(userRequest);
+                return MAPPER.writeValueAsBytes(resourceRequest);
             } catch (JsonProcessingException e) {
-                throw new SerializationFailedException("Failed to serialize " + userRequest.toString(), e);
+                throw new SerializationFailedException("Failed to serialise " + resourceRequest.toString(), e);
             }
         }
     }
 
     // Deserialiser for downstream test output
     static class ResponseDeserializer implements Deserializer<JsonNode> {
         @Override
-        public JsonNode deserialize(final String s, final byte[] userResponse) {
+        public JsonNode deserialize(final String s, final byte[] resourceResponse) {
```

Comment:
```suggestion
        public JsonNode deserialise(final String s, final byte[] resourceResponse) {
```

Reply:
cant rename method that we're overriding.  Java uses the american spelling

Reply:
I think I meant to get the class name but got distracted by the method name